src/lucky_migrator/create_table_statement.cr

```diff
@@ -58,25 +58,26 @@ class LuckyMigrator::CreateTableStatement
 
   # Generates raw sql from a type declaration and options passed in as named
   # variables.
-  macro add(type_declaration, index = false, using = :btree, unique = false)
+  macro add(type_declaration, index = false, using = :btree, unique = false, default = nil)
     {% if type_declaration.type.is_a?(Union) %}
-      add_column :{{ type_declaration.var }}, {{ type_declaration.type.types.first }}, optional: true
+      add_column :{{ type_declaration.var }}, {{ type_declaration.type.types.first }}, optional: true, default: {{ default }}
     {% else %}
-      add_column :{{ type_declaration.var }}, {{ type_declaration.type }}
+      add_column :{{ type_declaration.var }}, {{ type_declaration.type }}, default: {{ default }}
     {% end %}
 
     {% if index || unique %}
       add_index :{{ type_declaration.var }}, using: {{ using }}, unique: {{ unique }}
     {% end %}
   end
 
-  def add_column(name, type : (String | Time | Int32 | Int64 | Float | Bool).class, optional = false, reference = nil, on_delete = :do_nothing)
+  def add_column(name, type : (String | Time | Int32 | Int64 | Float | Bool).class, optional = false, reference = nil, on_delete = :do_nothing, default : String | Time | Int32 | Int64 | Float | Bool | Symbol | Nil = nil)
     rows << String.build do |row|
       row << "  "
       row << name.to_s
       row << " "
       row << column_type(type)
       row << null_fragment(optional)
+      row << default_value(type, default, name) unless default.nil?
```

Comment:
Quick note: If you remove the custom `raise` for Int32 and Int64 then you won't need to pass the name